.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /deploy
 /screenshots
+/src/examples-highlights
```

Comment:
Ahh I see, my bad. I was just moving the script over from the material.angular repo. It would make a lot more sense to use that gulp task. I'll make that change. I can definitely move the built folder to the root as well @kara 👍